src/workspaces/dashboard/CloudInformation.ts

```diff
@@ -113,6 +114,22 @@ const GoogleCloudInformation = (props: GoogleCloudInformationProps): ReactNode =
           },
         }),
       ]),
+      div({ style: { paddingBottom: '0.5rem' } }, [
+        h(
+          Link,
+          {
+            style: { margin: '1rem 0.5rem' },
+            ...newTabLinkProps,
+            onClick: () => {
+              void Metrics().captureEvent(Events.workspaceOpenedProjectInConsole, {
+                ...extractWorkspaceDetails(workspace),
+              });
+            },
+            href: `https://console.cloud.google.com/welcome?project=${googleProject}&authuser=${getTerraUser().email}`,
```

Comment:
It would be good to include the `&authuser=` in the new quota links as well.

Otherwise, users can run into this situation:
- They are on account A in GCP Console
- And account B in Terra
- When clicking the link, it opens with identity A and project B, which may be a permission denied

I definitely forgot about this issue, apologies for not including it in the story.